app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -88,24 +91,27 @@ function initReactions() {
 
           getCsrfToken()
             .then(sendFetch('reaction-creation', formData))
-            .then(function(response) {
+            .then(function (response) {
               if (response.status === 200) {
                 response.json().then(successCb);
               }
             });
         };
       }
-      var butts = document.getElementsByClassName('toggle-reply-form');
-      for (var i = 0; i < butts.length; i++) {
-        var butt = butts[i];
-        butt.onclick = function(event) {
+
+      butts = document.getElementsByClassName('toggle-reply-form');
+
+      for (let i = 0; i < butts.length; i++) {
+        const butt = butts[i];
```

Comment:
Changed to `const` to make `butt` block scoped and it also fixes the lint error.